routingtable/filter/term_condition.go

```diff
@@ -101,3 +101,29 @@ func (t *TermCondition) matchesLargeCommunityFilters(pa *route.Path) bool {
 
 	return false
 }
+
+func (t *TermCondition) equal(x *TermCondition) bool {
+	if len(t.routeFilters) != len(x.routeFilters) {
+		return false
+	}
+
+	if len(t.communityFilters) != len(x.communityFilters) {
+		return false
+	}
+
+	if len(t.largeCommunityFilters) != len(x.largeCommunityFilters) {
+		return false
+	}
+
+	for i := range t.routeFilters {
+		if !t.routeFilters[i].equal(x.routeFilters[i]) {
+			return false
+		}
+	}
+
+	// TODO: Compare community filters
```

Comment:
Lets create an issue after merge, to track this TODO

Reply:
Ack.

Reply:
https://github.com/bio-routing/bio-rd/issues/216